tests/unit/tests_high_five_base.cpp

```diff
@@ -1406,3 +1411,128 @@ BOOST_AUTO_TEST_CASE(HighFiveRecursiveGroups) {
     // Using root slash
     BOOST_CHECK(file.exist(std::string("/") + DS_PATH));
 }
+
+BOOST_AUTO_TEST_CASE(H5Easy_scalar)
```

Comment:
Please move the "H5Easy" tests to another source. `tests_high_five_base.cpp` is for core and is way too long already, should be refactored.